### PR TITLE
Add contribute section

### PR DESCRIPTION
**Pull request by jamierocks**
Running on https://www-sponge-staging.jamierocks.uk/

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -70,34 +74,67 @@ <h1 class="row text-center">Get Sponge</h1>
     </div>
 </section>
 
-<!-- Contact Section -->
+<!-- Contribute Section -->
 <section class="sponsors" id="sponsors">
```
Comment (progwml6) [inline templates/index.html]: change the class & id if this section is getting changed

Comment (jamierocks) [inline templates/index.html]: yeah - forgot about that :P

File: templates/index.html [hunk 2]
```diff
@@ -70,34 +74,75 @@ <h1 class="row text-center">Get Sponge</h1>
...
+                    Critique new plugin ideas and help people out with Sponge? Look no further than the
+                    <a href="https://forums.spongepowered.org/">Sponge Forums</a>.
+                    Powered by Discourse to ensure just the right balance between awesome ease of use and terrifying
+                    addictiveness.
+                </p>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-lg-4 text-center">
+                <a class="btn btn-outline" href="https://docs.spongepowered.org/master/en-GB/contributing/index.html"
```
Comment (Tzky) [inline templates/index.html]: I'd replace that link with `https://docs.spongepowered.org/en/contributing/index.html`

Comment (jamierocks) [inline templates/index.html]: Would be cool if there was a page like `https://docs.spongepowered.org/languageselect.html?/contributing/index.html` but I'll get to that :P

Comment (Tzky) [inline templates/index.html]: Well... if you find a good way to do that, PR it to the docs ;)

Besides that: i think we're good to go on this PR.

File: templates/index.html [hunk 3]
```diff
@@ -70,34 +74,75 @@ <h1 class="row text-center">Get Sponge</h1>
...
+                    detailed bug reports about it? Come get involved on GitHub!
+                </p>
+            </div>
+            <div class="col-lg-4 text-center">
+                <h1>Docs</h1>
+                <br>
+                <p>
+                    Coding isn't your thing but you love helping others, get a thrill from hammering out the perfect
+                    prose, or just love nit-picking other peoples' grammar? Our fearsome and frankly unstoppable
+                    documentation team will be delighted to induct you into their team of infamous brigands and
```
Comment (progwml6) [inline templates/index.html:104]: team is used twice in the sentence i'd change the second reference to something like ranks

File: templates/index.html (hunk 2 above)
Comment (jamierocks) [inline templates/index.html]: Yeah - something for my todo list :P

**Pull request merged**